doc/py3-cmd.rst

```diff
@@ -18,16 +18,16 @@ To refresh all modules use the ``all`` keyword.
 
 .. code-block:: shell
 
-    # refresh any wifi modules
+    # refresh wifi modules
```

Comment:
module

Reply:
Plural. Add many as you want... It'll refresh all of them.

Reply:
sure, but that's not what the example below shows, so in this case you only refresh the wifi module